presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -212,6 +212,42 @@ public static Slice sha512(@SqlType(StandardTypes.VARBINARY) Slice slice)
         return Slices.wrappedBuffer(Hashing.sha512().hashBytes(slice.getBytes()).asBytes());
     }
 
+    @Description("compute HMAC MD5 hash")
```

Comment:
Let's add these in a new class instead of cramming more stuff into VarbinaryFunctions. You can name the class HmacFunctions.

Reply:
I think the description should be “compute MD5 HMAC” since HMAC stands for “hash message authentication code” and thus saying “hash” is redundant.

Reply:
Okay, will update.